layouts/default.vue

```diff
@@ -10,41 +10,100 @@
           </NuxtLink>
         </div>
 
-        <!-- Search Bar -->
-        <div class="flex-1 mx-4">
-          <input
-            type="text"
-            placeholder="Search for products..."
-            class="w-full px-4 py-2 rounded-md text-gray-700 outline focus:outline-none focus:ring-2 focus:ring-blue-500"
-          />
-        </div>
-
-        <!-- Icons -->
-        <div class="flex items-center space-x-4">
-          <!-- Wishlist -->
-          <NuxtLink to="/wishlist" class="hover:text-blue-400">
-            <i class="fas fa-heart"></i> Wishlist
-          </NuxtLink>
-          <!-- Cart -->
-          <NuxtLink to="/cart" class="hover:text-blue-400">
-            <i class="fas fa-shopping-cart"></i> Cart
-          </NuxtLink>
-          <!-- Join Business -->
-          <NuxtLink to="/joinbusiness" class="hover:text-blue-400">
-            <i class="fas fa-briefcase"></i> Join Business 
-          </NuxtLink>
-          <p>|</p>
-          <!-- Sign In -->
-          <NuxtLink to="/signin" class="hover:text-blue-400">
-            <i class="fas fa-user"></i> Sign In
-          </NuxtLink>
-        </div>
+        <!-- Navigation header -->
+        <nav class="flex-1 mx-4">
+          <ul class="flex items-center space-x-6">
+            <!-- Search Bar -->
+            <li class="flex-1">
+              <input

```

Comment:
Please move to use EL-Plus input (el-input) for easy handle the search or input data.